vllm/model_executor/models/bloom.py

```diff
@@ -109,8 +109,8 @@ def __init__(
         scaling = self.head_dim**-0.5
         self.attn = PagedAttention(self.num_heads,
                                    self.head_dim,
-                                   scaling,
-                                   alibi_slopes=alibi_slopes)
+                                   scaling)#,
+                                   #alibi_slopes=alibi_slopes)
```

Comment:
this should be removed

Reply:
do you mean restoring the original version?

Reply:
yes

Reply:
fixed